### PR TITLE
Update 'async_add_pumping' for BB v2 API

### PR DESCRIPTION
**Pull request by seanhagen**

### CODE REVIEW

File: custom_components/babybuddy/switch.py [hunk 1]
```diff
@@ -248,6 +257,32 @@ async def async_add_sleep(
...
+    ) -> None:
+        """Add a pumping entry."""
+        try:
+            data = self.set_common_fields(timer, start, end)
+        except ValidationError as err:
+            _LOGGER.error(err)
+            return
+
+        data.update( {
+            ATTR_CHILD: self.child[ATTR_ID],
```
Comment (jcgoette) [inline custom_components/babybuddy/switch.py:276]: You're using `COMMON_FIELDS`, so shouldn't need to set `ATTR_CHILD` here.

**Comment by jcgoette** [comment]
Thanks for taking a swing at this!

Make sure to `black` your code.

**Comment by seanhagen** [comment]
No problem! Python is similar enough to Go I felt I could take a stab at it 😅

**Comment by jcgoette** [comment]
@seanhagen have you had a chance to review again?

**Pull request closed without merging**